package.json

```diff
@@ -10,9 +10,14 @@
     "build": "npm run lint && npm run compile",
     "prepublish": "npm run build",
     "lint": "eslint src",
-    "test": "echo \"To be implemented\"",
+    "test": "better-npm-run test",
     "compile": "babel src --out-dir dist"
   },
+  "betterScripts": {
```

Comment:
Just as a reminder, we won't need this after you change everything to use SQLite @evasquez26